core/src/duplicate_repair_status.rs

```diff
@@ -350,24 +336,28 @@ impl AncestorRequestStatus {
                         return DuplicateAncestorDecision::InvalidSample;
                     }
                     (
-                        Some((mismatch_i, DuplicateAncestorDecision::EarliestAncestorNotFrozen(_))),
+                        Some((
```

Comment:
This arm is now only relevant for pruned pathway where we don't have hashes to compare